app/controllers/user/mygardens/plants_controller.rb

```diff
@@ -22,13 +26,17 @@ def destroy
   
   def add_notification(garden, plant)
     title = plant.name.capitalize + " harvest time!"
-    description = "It's time to harvest your #{plant.name}"
+    description = "It's time to harvest your #{plant.name} in #{garden.name}"
```

Comment:
specifies which `garden` the `notification` is for so that `users` can distinguish the `notifications` when the same `plant` is added to different `gardens`